packages/transmute-dashboard/src/transmute-config/env.json

```diff
@@ -10,10 +10,7 @@
       "protocol": "http"
     },
     "web3Config": {
-      "providerUrl": "http://localhost:8545",
-      "host": "localhost",
-      "port": 8545,
-      "protocol": "http"
+      "providerUrl": "http://localhost:8545"
```

Comment:
This should be switched to port 9545, if we are going to be pushing use of ganache.
http://truffleframework.com/docs/getting_started/console - just do a search for that port.